xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -209,6 +209,17 @@
       "channels": { "type": "integer", "required": true }
     }
   },
+  "Player.Video.Stream": {
+    "type": "object",
+    "properties": {
+      "index": { "type": "integer", "minimum": 0, "required": true },
+      "name": { "type": "string", "required": true },
+      "language": { "type": "string", "required": true },
```

Comment:
We can't change those because it would break the API. But I didn't realize it was the same there. In that case we can leave it like this to be consistent.